app/api/favorites.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from fastapi import Depends
+from fastapi import Query
+
+import app.usecases
+from app.models.user import User
+from app.usecases.user import authenticate_user
+
+
+async def add_favorite(
+    user: User = Depends(authenticate_user(Query, "u", "h")),
```

Comment:
Confirmed params via fiddler

Reply:
Sorry for no image i closed it

Reply:
https://github.com/osuAkatsuki/bancho.py/blob/e8817c8addf733e1e0b873f7b66aabd41cb98254/app/api/domains/osu.py#L247-L262